Automattic/jetpack
# Add new fonts

Adding a set of new fonts.

See p9Jlb4-5Dj-p2 for design discussion.

![image](https://user-images.githubusercontent.com/87168/205605485-28292e7d-8579-44c1-b9b0-3cddd8674500.png)


#### Changes proposed in this Pull Request:

- Cleans up the list a bit (" to ' and alphabetical order)
- Adds new fonts:

**Sans Serif fonts**
* [Albert Sans](https://fonts.google.com/specimen/Albert+Sans)
* [Commissioner](https://fonts.google.com/specimen/Commissioner)
* [Plus Jakarta Sans](https://fonts.google.com/specimen/Plus+Jakarta+Sans)
* [Epilogue](https://fonts.google.com/specimen/Epilogue)
* [Sora](https://fonts.google.com/specimen/Sora)
* [Figtree](https://fonts.google.com/specimen/Figtree)

**Serif Fonts**
* [Piazzola](https://fonts.google.com/specimen/Piazzolla)
* [Alegreya](https://fonts.google.com/specimen/Alegreya)
* [Cormorant](https://fonts.google.com/specimen/Cormorant)
* [Petrona](https://fonts.google.com/specimen/Petrona)
* [Crimson Pro](https://fonts.google.com/specimen/Crimson+Pro)
* [Fraunces](https://fonts.google.com/specimen/Fraunces)

**Monospace font**
* [DM Mono](https://fonts.google.com/specimen/DM+Mono)


Unlike in the screenshot above and the P2, these two fonts were omitted as they aren't Google Fonts:
* Uncut Sans
* Metropolis

We'll need to look at how to add them separately.


#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* In site editor, go to "global styles" panel and change fonts. See new fonts available
